apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/DocumentDeleteResource.java

```diff
@@ -158,7 +158,10 @@ public Uni<RestResponse<Object>> deleteSubDocument(
       @QueryParam("profile") boolean profile) {
     ExecutionContext context = ExecutionContext.create(profile);
     List<String> subPath =
-        documentPath.stream().map(PathSegment::getPath).collect(Collectors.toList());
+        documentPath.stream()
+            .map(PathSegment::getPath)
+            .filter(p -> !p.isEmpty())
+            .collect(Collectors.toList());
```

Comment:
Not a big deal but since it's called from a few resources, could also extract into helper method.